workbench-core/datasets/src/dataSetService.ts

```diff
@@ -50,16 +52,30 @@ export class DataSetService {
    * @returns the DataSet object which is stored in the backing datastore.
    */
   public async provisionDataSet(request: CreateProvisionDatasetRequest): Promise<DataSet> {
-    const { storageProvider, ...dataSet } = request;
-
-    await storageProvider.createStorage(dataSet.storageName, dataSet.path);
-
-    const provisioned: DataSet = {
-      ...dataSet,
-      storageType: storageProvider.getStorageType()
+    const metadata: Metadata = {
```

Comment:
will also need to include an actor

Reply:
Not on this PR, but will come in when this method is updated to add an access permission.